src/components/SwapCatalogue/SwapCatalogue.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from "react";
+import styles from "./SwapCatalogue.module.scss";
+import sample from "../../images/Sample.svg";
+import { ThemeContext } from "../../App";
+import { useNetwork } from "wagmi";
+
+function SwapCatalogue(props) {
+  const themes = useContext(ThemeContext);
+  const { theme } = themes;
+
+  const swapAction = () => {
+    props.setOpen(false);
+    props.setSwap(true);
+    const nft = props.nft;
+    nft.hasSelected = true;
+    nft.selectedNFT_approve_count = 1;
+    nft.selectedNFT_swap_count = 1;
+    props.setSelected(nft);
+    props.setIndex(props.index + 1);
+  };
+  const { chain } = useNetwork();
+  let image_url = "";
+
+  try {
+    if (chain.network === "Godwoken ") {
```

Comment:
the space after Godwoken can cause unexpected bugs; please review

Reply:
Reviewed, its all good here!